src/main/java/graphql/nadel/engine/NadelContext.java

```diff
@@ -67,12 +69,16 @@ public NormalizedQueryFromAst getNormalizedOverallQuery() {
         return normalizedOverallQuery;
     }
 
+    public boolean getOptimizeExecutionFlag() {
+        return nadelExecutionHints.getOptimizeExecutionFlag();
+    }
```

Comment:
I think this should stay in the hints class and just call it via the Hints object

Reply:
I agree - and also not need for the `Flag` suffix.